.changeset/green-lies-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/exchange-persisted-fetch': minor
+---
+
+Adds support for custom hash functions by adding a `generateHash` option in the `createPersistedFetch` factory.
```

Comment:
Sorry, just a small nit here. It says “in the `createPersistedFetch` factory”, but it should probably just say something like “to the exchange”, right?

It’s a slightly important nit since the changesets are combined into the next CHANGELOG entries for each release, so they’ll need to be clear and descriptive

Reply:
Ah, good call 👍